flow/server/flow_subscriber_endpoint.go

```diff
@@ -29,7 +29,23 @@ type FlowSubscriberEndpoint struct {
 
 // SendFlows sends flow to the subscribers
 func (fs *FlowSubscriberEndpoint) SendFlows(flows []*flow.Flow) {
-	msg := ws.NewStructMessage("flow", "store", flows)
+	fs.sendFlows("flow", flows)
+	flowsByCaptureMap := make(map[string][]*flow.Flow)
```

Comment:
It would be better to do things only when we do have a subscriber for a specific namespace. I will check how to handle this otherwise sounds good. BTW I submitted a PR to fix broadcast Namespace filter
https://github.com/skydive-project/skydive/pull/1949

Reply:
i guess skipping when we have no subscribers is an optimization which is orthogonal to this PR - right?

Reply:
Yes that something we can address later. That said please have a look at this
https://github.com/skydive-project/skydive/pull/1949/commits/36fab3b5cbe6028c1140004c578183471e64476e
just added the mechanism

Reply:
ok (see review comment)